src/app/core/auth/auth.resolver.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  Resolve,
+  ActivatedRouteSnapshot,
+  RouterStateSnapshot
+} from "@angular/router";
+import { Observable } from "rxjs";
+import { Injectable } from "@angular/core";
+import { AuthService } from "./auth.service";
+import { catchError } from "rxjs/operators";
+
+@Injectable()
+export class AuthResolver implements Resolve<any> {
```

Comment:
is this resolver needed as the same check is being done by the `initializeApplication()` within `auth.initalizer.ts`?

Reply:
Short answer: No. Long answer. If authenticated and role !== RevalAdmin we'd need to use new route e.g. error/notauthourised etc. We can delete as its not used / imported anywhere but guaranteed we'd write this again. Hope that makes sense?

Reply:
ah im with you now. thanks for explaining. as discussed lets take this out for now and tackle once we implement roles logic

Reply:
deleted as requested